Sources/StreamChat/APIClient/Endpoints/Payloads/PollsPayloads.swift

```diff
@@ -0,0 +1,251 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct PollPayload: Decodable {
+    let duration: String
+    let poll: PollResponseData
+
+    init(duration: String, poll: PollResponseData) {
```

Comment:
Probably it would make more sense to switch the naming here. The higher level type is the Response, and the other is the actual payload. We have this same pattern in other places. I feel that `ResponseData` is a bit too generic to describe the structure of a type. Especially the `Data` post-fix, which we use sometimes if the type is binary.
```suggestion
struct PollPayloadResponse: Decodable {
    let duration: String
    let poll: PollPayload

    init(duration: String, poll: PollPayload) {
```